Backend/purrfect-match-backend/src/main/java/com/purrfectmatch/restservice/LoadDatabase.java

```diff
@@ -0,0 +1,59 @@
+package com.purrfectmatch.restservice;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.sql.Date;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@Slf4j
+class LoadDatabase {
+        @Bean
+        CommandLineRunner initDatabase(ShelterAccountRepository shelterAccountRepository,
+                        SuccessStoryRepository successStoryRepository,
+                        RecentlyAdoptedRepository recentlyAdoptedRepository, BreedRepository bRepo) {
+                return args -> {
+                        /*
+                         * log.info("Preloading " + bRepo.save(new Breed("spaniel", Species.dog)));
+                         * log.info("Preloading " + bRepo.save(new Breed("german shepherd",
+                         * Species.dog))); log.info("Preloading " + bRepo.save(new Breed("dachshund",
+                         * Species.dog))); log.info("Preloading " + bRepo.save(new
+                         * Breed("saint bernard", Species.dog))); log.info("Preloading " +
+                         * bRepo.save(new Breed("husky", Species.dog))); log.info("Preloading " +
+                         * bRepo.save(new Breed("golden retriever", Species.dog)));
+                         * log.info("Preloading " + bRepo.save(new Breed("pug", Species.dog)));
+                         * log.info("Preloading " + bRepo.save(new Breed("french bulldog",
+                         * Species.dog))); log.info("Preloading " + bRepo.save(new Breed("pitbull",
+                         * Species.dog))); log.info("Preloading " + bRepo.save(new Breed("chihuahua",
+                         * Species.dog))); log.info("Preloading " + bRepo.save(new Breed("great dane",
+                         * Species.dog))); log.info("Preloading " + bRepo.save(new Breed("pomerian",
+                         * Species.dog)));
+                         */
```

Comment:
Probably doesn't need to be block commented, this is useful test data to have for everyone.